src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -263,5 +263,10 @@
         <service id="Sylius\Component\Core\Cart\Resolver\CreatedByGuestFlagResolverInterface" class="Sylius\Component\Core\Cart\Resolver\CreatedByGuestFlagResolver">
             <argument type="service" id="security.token_storage" />
         </service>
+
+        <service id="sylius.complete_state_guard" class="Sylius\Component\Core\StateGuard\CompleteStepGuard" />
```

Comment:
```suggestion
        <service id="sylius.guard.complete_state" class="Sylius\Component\Core\StateGuard\CompleteStepGuard" />
```
imo it is more readable name : `sylius.type_of_service.detail`

Reply:
sylius.state_guard.address/complete etc work for you?